packages/util/src/declarations/snarkjs.d.ts

```diff
@@ -2,7 +2,7 @@
 declare module "snarkjs" {
   namespace groth16 {
     async function fullProve(
-      input: Record<string, string>,
+      input: Record<string, string | string[]>,
```

Comment:
@cedoor - if the typing here is incorrect it may also be incorrect in the typings you've created for #465 - could you check?

Reply:
I'm still working on `@types/snarkjs`, I'll use something like https://github.com/erhant/circomkit/blob/main/src/types/circuit.ts, which look well defined. `@types/circomlibjs` is ready.

PRs: [`circomlibjs`](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66676), [`snarkjs`](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66703)